mllib/src/main/scala/org/apache/spark/mllib/clustering/LDAModel.scala

```diff
@@ -361,6 +361,43 @@ class DistributedLDAModel private (
     }
   }
 
+  /**
+   * Return the top documents for each topic
+   *
+   * This is approximate; it may not return exactly the top-weighted documents for each topic.
```

Comment:
Btw, I had thought this originally, but is this actually approximate?  I think it's exact.  But I'm pretty tired, so it would be great to get confirmation from you @rotationsymmetry and @hhbyyh